app/assets/stylesheets/scaffold.css

```diff
@@ -0,0 +1,84 @@
+body {
```

Comment:
This file is autogenerated by:
`rails g scaffold User username:string email:string`
so I don't know how to get it into the gulp pipeline